collections/Ash_Rolls/MS_Ash_Rolls_38.xml

```diff
@@ -74,7 +74,7 @@
                      </supportDesc>
                   </objectDesc>
                   <decoDesc>
-                     <decoNote resp="#PA" type="miniature">Good miniatures (coloured drawings), borders. (P&amp;A iii. 535)</decoNote>
+                     <decoNote resp="#PA" type="miniature">Good miniatures (coloured drawings), borders. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 535)</decoNote>
```

Comment:
Not sure whether it is necessary to have a link here as well as in the bibliography.